pkg/network/cache/multi.go

```diff
@@ -192,6 +193,13 @@ func (x *multiClient) ReportError(err error) {
 		return
 	}
 
+	x.addrMtx.RLock()
+	group := x.addr
```

Comment:
`x.addr` can be already unrelated to the `err` at this moment and, if so, we'll see a lie in the log

Reply:
why is it a problem? we are invalidating clients currently associated with _this_ address group

added the exact problem address to the err on the higher level

Reply:
u fixed message and now it conveys a fact

Reply:
> is it guaranteed that the x.addr here exactly related to the x.clients below?

reorganized mutexes a little, now it can be